backend/app/store/user.go

```diff
@@ -15,14 +15,15 @@ import (
 
 // User holds user-related info
 type User struct {
-	Name     string `json:"name"`
-	ID       string `json:"id"`
-	Picture  string `json:"picture"`
-	IP       string `json:"ip,omitempty"`
-	Admin    bool   `json:"admin"`
-	Blocked  bool   `json:"block,omitempty"`
-	Verified bool   `json:"verified,omitempty"`
-	SiteID   string `json:"site_id,omitempty"`
+	Name              string `json:"name"`
+	ID                string `json:"id"`
+	Picture           string `json:"picture"`
+	IP                string `json:"ip,omitempty"`
+	Admin             bool   `json:"admin"`
+	Blocked           bool   `json:"block,omitempty"`
+	Verified          bool   `json:"verified,omitempty"`
+	EmailSubscription bool   `json:"email_subscription,omitempty"`
```

Comment:
unless I missed it, I don't see any place using or testing this new EmailSubscription flag.

Reply:
https://github.com/umputun/remark/pull/528/files#diff-8c75f8c35267cecc93710ac2ec291accR211, I'll add a test.

Reply:
Added in 4e00428.